lib/mfa/hotp.js

```diff
@@ -4,36 +4,34 @@
  * Handles Multi Factor Authentication for accessing parts of the
  * application we want to verify the client has access to.
  *
+ * We want to promisfy all methods we can because some of them
+ * take some time to generate and we don't want code going on
+ * till we have the data we need back.
+ *
+ * @param {Object} config
+ * @param {Object} twofa
+ * @param {Object} promise
+ * @return {Object} hotp object
```

Comment:
The words on top are great.  The explanation of the paramters are pretty much not great.  Please remove per our earlier discussion.